assets/main.js

```diff
@@ -0,0 +1,311 @@
+import Alpine from 'https://cdn.jsdelivr.net/npm/alpinejs@3/dist/module.esm.js'
```

Comment:
Personally I'm not a big fan of CDNs regarding availability, security and data (GDPR).

Reply:
I thought build system was too much for this and went with cdn's.

Reply:
Can we find a good and effective way to use alpine without CDN and make the build setup smaller?
I'm open for recommendations and ideas.

If the CDN solution is the best solution and others are ok with this, then we can maybe make this step.

In my personal and other projects I've tried to add these things, same for example in materialize: https://github.com/materializecss/materialize/tree/v1-dev/docs/js (the dependencies are shipped with the website).

Reply:
We can maybe add the alpinejs file (as fallback) if someone wants to selfhost the web version of the minifier (and can not use CDNs in the corporate network due to internal policies) or if something may happen with the used CDN domains (just an idea so in the worst case we can then switch to the included version).

Reply:
For now I used skypack, they are GDPR compliant.

May be I can download alpinejs to assets dir and just use. Later with rollup, may be I can try making alpine a devDependency.

Reply:
> In my personal and other projects I've tried to add these things, same for example in materialize: https://github.com/materializecss/materialize/tree/v1-dev/docs/js (the dependencies are shipped with the website).

can be done.

Reply:
Just a thought. I agree to all your concerns about cdn's as they are 100% valid. but personally I feel for Github pages and I wouldn't worry much about it.

Reply:
Shall I go ahead and merge this? may be if someone complains we can take the other fallback options available.

Reply:
You can go ahead. I'm fine with the current changes.